orchestra/models/core/models.py

```diff
@@ -607,6 +646,12 @@ class Meta:
     completed = models.BooleanField(default=False)
     start_by_datetime = models.DateTimeField(null=True, blank=True)
     due_datetime = models.DateTimeField(null=True, blank=True)
+    skipped = models.BooleanField(default=False)
```

Comment:
skipped_datetime so we can reconstruct when these happened :)

Reply:
@marcua A few doubts. :)
- Arent we planning to use activity log to reconstruct what happened?
- What happens when a user skips and then adds back? I suppose we set the field to null when someone unskips a todo.

Reply:
Cool :) I am a big fan of Google analytics event tags too :)